numba/blackscholes/GPU/bs_erf_numba_jit_par.py

```diff
@@ -4,11 +4,22 @@
 
 import dpctl
 import base_bs_erf
-import numba as nb
+from device_selector import get_device_selector
+
 from math import log, sqrt, exp, erf
 
+backend = os.getenv("NUMBA_BACKEND", "legacy")
+if backend == "legacy":
```

Comment:
Can we move this `if backend` boilerplate to some common place? into `dpbench_decorators.py` for example

Reply:
unfortunately not yet. We'll also think of some way to pass gpu/cpu selector as well